bigquery/api/README.rst

```diff
@@ -0,0 +1,395 @@
+.. This file is automatically generated. Do not edit this file directly.
+
+Google BigQuery Python Samples
+===============================================================================
+
+This directory contains samples for Google BigQuery. `BigQuery`_ is Google's fully managed, petabyte scale, low cost analytics data warehouse. BigQuery is NoOps—there is no infrastructure to manage and you don't need a database administrator—so you can focus on analyzing data to find meaningful insights, use familiar SQL, and take advantage of our pay-as-you-go model.
```

Comment:
The rendered 'BigQuery_' in the second sentence here seems to appear as a link that goes nowhere (and the trailing underscore still appears) (ditto below)